ss_rules.json

```diff
@@ -0,0 +1,792 @@
+{
+	"Section-Ad-Card (sac)": {
+		"section_type":	"frontmatter",
+		"contiguous_block_criteria_01": {
+			"multiple":	"False",
```

Comment:
GLOBAL: change all "True" and "False" values to "true" and "false" (it's more standard).